### DISCUSSION TITLE
v2.5.1

### DISCUSSION THREAD

**Discussion by Zenda-Cross** [vega-org/vega-app#141] [Announcements]
- Provider improved Vegamovies,MoviesApi,AnimeRulz and MultiStream.
- Volume and Brightness gestures (internal player)
- Double tap to seek gestures (internal player)
- Option to hide seek buttons (settings)

<hr /><em>This discussion was created from the release <a href='https://github.com/Zenda-Cross/vega-app/releases/tag/v2.5.1'>v2.5.1</a>.</em>

**Comment by adilurrhmn** [comment]
Please fix the providers - showing blank screen

LuxMovies
Hdhub4u
Dooflix
Netflix mirror
Multistream - change the server, hindi dubbed audio are not playble
Extramovies - you can remove this providers( they uses vegamovies and LuxMovies contents)

**Reply by Zenda-Cross** [reply]
I tried dooflix is working isn't it ?
I will fix other providers

**Comment by adilurrhmn** [comment]
Yes, dooflix is working now
